ui-v2/app/templates/components/resolver-card.hbs

```diff
@@ -0,0 +1,46 @@
+<div class="resolver-card">
+  <header onclick={{onclick}} id={{concat 'resolver:' item.ID}}>
+    <a name="">
+      <h3>{{item.Name}}</h3>
+{{#if item.Failover}}
+      <dl>
+        <dt data-tooltip={{concat item.Failover.Type ' failover'}}>{{concat item.Failover.Type ' failover'}}</dt>
+        <dd>
+          <ol>
+  {{#each item.Failover.Targets as |item|}}
+            <li>
+              <span>{{item}}</span>
+            </li>
+  {{/each}}
+          </ol>
+        </dd>
+      </dl>
+{{/if}}
+    </a>
+  </header>
+{{#if (gt item.Subsets.length 0)}}
+    <ul>
+  {{#each item.Subsets as |item|}}
+      <li onclick={{onclick}} id={{concat 'resolver:' item.ID}}>
+        <a name="">
```

Comment:
I think you can add keyboard navigability/accessibility to this pretty easily by changing to `buttons`. 

I made a codepen to test my own assumptions. Try tabbing through and hitting enter and space on stuff. https://codepen.io/DingoEatingFuzz/pen/gObMPoz

Reply:
See above comments re: anchor/checkboxes.

I also had a feeling that buttons have a restriction over what you should put inside them - I wasn't sure about this as it was just a feeling, but I've had a quick look at it seems like you shouldn't put block elements inside a `button`, so that would severely restrict us here.

The only other thing would be to do some absolute positioning of a button over the rest of the element - but I really don't want to go there and seem like that would create more problems than we are trying to solve here.

Reply:
Ha, sounds like the toggle I just implemented for node drain. Uses a hidden checkbox for accessibility/keyboard and is wrapped in a `label` that contains the richer markup.

Reply:
Yes, exactly!

Reply:
I was curious as to whether this was an old thing I was remembering (like for a's) or if the "don't put block things inside buttons" was just some feeling I had, so I took a little bit of a deeper look:

https://developer.mozilla.org/en-US/docs/Web/HTML/Element/button

> Permitted content: Phrasing content but there must be no Interactive content

https://developer.mozilla.org/en-US/docs/Web/Guide/HTML/Content_categories#Phrasing_content

So we can't use `dl`'s, `ol`s etc in buttons, which we use here.

I think generally I'd continue to steer clear of using buttons to wrap anything that could potentially end up being anything more than just a piece of text.

Reply:
😮 

> not only consisting of white spaces characters

I'm sure we do this somewhere 😆 ! Well I think we have empty buttons somewhere, so not whitespace as such, but just empty - might check on that further down the line!